pkg/operator/ceph/cluster/controller.go

```diff
@@ -215,15 +218,20 @@ func add(mgr manager.Manager, r reconcile.Reconciler, context *clusterd.Context)
 // Result.Requeue is true, otherwise upon completion it will remove the work from the queue.
 func (r *ReconcileCephCluster) Reconcile(context context.Context, request reconcile.Request) (reconcile.Result, error) {
 	// workaround because the rook logging mechanism is not compatible with the controller-runtime logging interface
-	reconcileResponse, err := r.reconcile(request)
+	reconcileResponse, cephCluster, err := r.reconcile(request)
+
 	if err != nil {
 		logger.Errorf("failed to reconcile. %v", err)
+		r.clusterController.recorder.Report(cephCluster, corev1.EventTypeWarning, "ReconcileFailed", err.Error())
+	} else {
+		logger.Debug("reconcile succeeded.")
+		r.clusterController.recorder.Report(cephCluster, corev1.EventTypeNormal, "ReconcileSucceed", "cephCluster has been configured successfully")
```

Comment:
For consistency with "ReconcileFailed"
```suggestion
		r.clusterController.recorder.Report(cephCluster, corev1.EventTypeNormal, "ReconcileSucceeded", "cluster has been configured successfully")
```

Reply:
Done